fixtures/cmr.uat.earthdata.nasa.gov-443/165099102895369026

```diff
@@ -0,0 +1,25 @@
+POST /search/clear-scroll
```

Comment:
I'm not sure where this fixture came from, but it kept getting generated for me. Running mocha now I don't see any new fixtures generated.